### DISCUSSION TITLE
Episode 133: Lambda Days Live! José Valim, Alexis King & More!

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#24] [Episode]
Episode Link: https://adspthepodcast.com/2023/06/09/Episode-133.html

Feel free to write comments or ask questions :slightly_smiling_face:

**Comment by filiplajszczak** [comment]
link to  Toby Pfeiffer's talk https://youtu.be/WU5u2x_5QLA could be added to notes as promised.

**Reply by codereport** [reply]
Thanks! Updated 🙏
